src/main/drivers/pwm_output.c

```diff
@@ -141,7 +141,13 @@ static void pwmWriteOneshot(uint8_t index, uint16_t value)
 
 static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 {
-    *motors[index]->ccr = 60001 * (value - 1000) / 250000 + 60;
+
+#if defined(STM32F40_41xxx) || defined(STM32F411xE)
+    *motors[index]->ccr = (uint16_t)((float)(value-1000) / 1.04166f)+ 240;
```

Comment:
100 x ((looper2-1000)+240) took 11µs and gives 96% accuracy.
100 x (60001 * (value - 1000) / 62500 + 240;) took 16µs
100 x (*motors[index]->ccr = (uint16_t)((float)(value-1000) / 1.04166f)+ 240;) took 25µs